app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 80px;
+  background-color: #000;
+  color: #fff;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  font-size: 12px;
+
+  /* Position the tooltip text */
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  left: 50%;
+  margin-left: -43px;
+
+  /* Fade in tooltip */
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+
+/* Tooltip arrow */
+.tooltip .tooltiptext::after {
```

Comment:
Not sure about what this does (I suppose it hides some artifact element), but `content: "none";` should be sufficient here, as it removes the pseudoelement at all

Reply:
To me it creates a small element which uses the following hack for displaying a triangle: https://css-tricks.com/snippets/css/css-triangle/ - I would leave it as it is (it works). Maybe I will leave this CSS triangle link in a comment